smart_open/tests/test_azure.py

```diff
@@ -257,20 +280,9 @@ def test_get_blob_client(self):
 
 
 if DISABLE_MOCKS:
-    """If mocks are disabled, allow to use the Azurite local Azure Storage API
-    https://github.com/Azure/Azurite
-    To use locally:
-    docker run -p 10000:10000 -p 10001:10001 mcr.microsoft.com/azure-storage/azurite
-    """
-    # use Azurite default connection string
-    CONNECT_STR = 'DefaultEndpointsProtocol=http;' \
-        'AccountName=devstoreaccount1;' \
-        'AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/' \
-        'K1SZFPTOtr/KBHBeksoGMGw==;' \
-        'BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;'
     test_blob_service_client = azure.storage.blob.BlobServiceClient.from_connection_string(CONNECT_STR)
 else:
-    test_blob_service_client = FakeBlobServiceClient()
+    test_blob_service_client = FakeBlobServiceClient.from_connection_string(CONNECT_STR)
```

Comment:
This long variable name is making a lot of formatting look awkward - you're splitting simple statements over multiple lines just because of it. Use something shorter, e.g.

```suggestion
    CLIENT = FakeBlobServiceClient.from_connection_string(CONNECT_STR)
```